Formula/autopano-sift-c.rb

```diff
@@ -22,10 +22,20 @@ class AutopanoSiftC < Formula
     sha256 cellar: :any, yosemite:    "f38fa9a0dc3b30352155bafdad91f18b01ddc11db7c27c164d23def252ec7513"
   end
 
+  # Upstream dropped support due to unclear licensing issues see:
+  #   https://groups.google.com/g/hugin-ptx/c/85yJ6SSd7Eo/m/SkxWF3hGBQAJ
+  # Last update was in 2009
+  disable! date: "2021-03-28", because: :unmaintained
```

Comment:
I believe this could be `incompatible_license` instead?

Reply:
I think we only have `:no_license` as a license-related [disable reason](https://github.com/Homebrew/brew/blob/7ae9cc346788ab7a022f033df08943df52e60273/Library/Homebrew/deprecate_disable.rb#L10-L19).
```ruby
  DEPRECATE_DISABLE_REASONS = {
    does_not_build:      "does not build",
    no_license:          "has no license",
    repo_archived:       "has an archived upstream repository",
    repo_removed:        "has a removed upstream repository",
    unmaintained:        "is not maintained upstream",
    unsupported:         "is not supported upstream",
    deprecated_upstream: "is deprecated upstream",
    versioned_formula:   "is a versioned formula",
  }.freeze
```

Reply:
👍